template/build/vue-loader.conf.js

```diff
@@ -8,5 +8,9 @@ module.exports = {
       ? config.build.productionSourceMap
       : config.dev.cssSourceMap,
     extract: isProduction
-  })
+  }),
+  transformToRequire: {
```

Comment:
Can you keep it 2 spaces and add the default values as well: ` { img: 'src', image: 'xlink:href' }`